tests/src/item.test.ts

```diff
@@ -0,0 +1,398 @@
+import * as anchor from "@project-serum/anchor";
+import { Transaction } from "@raindrop-studios/sol-kit";
+import { ItemProgram, Instructions } from "@raindrops-protocol/raindrops";
+import { IDL as ItemProgramIDL } from "./types/item";
+import { createMintNFTInstructions } from "./utils/token";
+
+describe("Item Program", () => {
+  it.concurrent("Item Flow", async () => {
+    const walletKeypair = anchor.web3.Keypair.generate();
+
+    const provider = new anchor.AnchorProvider(
+      new anchor.web3.Connection("http://localhost:8899"),
+      new anchor.Wallet(walletKeypair),
+      { commitment: "confirmed" }
+    );
+
+    await provider.connection.requestAirdrop(
+      walletKeypair.publicKey,
+      100 * anchor.web3.LAMPORTS_PER_SOL
+    );
```

Comment:
Using Anchor, all of this stuff can be done for us. A provider is automatically setup and given lamports sufficient for operations

Reply:
https://github.com/raindrops-protocol/raindrops/blob/master/rust/tests/namespace.ts#L17 for ref^

Reply:
Of course, we can use anchor provider but sometimes, we need to use different wallets in parallel tests.
Especially in jest, all tests are running in parallel, so we should avoid wallet conflicting.

Reply:
> 

I think we should just move these into the rust directory as these feel like integration tests more than system level tests anyway.

I also dont think running in parallel is necessarily a good idea when running a lot of tests against the local validator, it can easily fall behind confirming blocks and you get cascading failures. IMO we shouldnt try to make things run in parallel right now, we should focus on writing solid tests and then run them in parallel if we feel the need to optimize, its just premature right now

Reply:
I see, make sense, I will reorganize the system-wide tests and unit tests.